### ISSUE TITLE
Precompiled binary

### ISSUE THREAD

**Issue by cnlohr** [cnlohr/cnping#65]
@dreua do you have any plans for how we would distribute cnping.exe?  I am temporarily removing it from the .gitignore file, but we should find a better way to distribute, sign, etc.

**Comment by cnlohr** [comment]
I've noticed this site: https://www.certum.eu/en/cert_offer_en_open_source_cs/ but I'm not sure if I need it or how I would go about it.

**Comment by dreua** [comment]
I believe I proposed a solution in  #32 . Signing should be possible via travis as well (it's essentially a linux environment), might need a bit of work to set it up though.

**Comment by dreua** [comment]
The Travis build is running smoothly :) About the signing: I'm not sure if it is worth the money, to be honest. I just read some stuff on the web that didn't sound too promising. I will ask a college about his experience on that next week.

**Comment by dreua** [comment]
I there something else to be done here? Github actions became a thing in the meantime and I think they are somewhat better integrated than travis, but travis still works so I think we can leave it as is for now.

**Comment by dreua** [comment]
Okay CI is currently broken again, I don't think this is our fault: https://travis-ci.com/github/cntools/cnping

I'm thinking about getting rid of Travis in favor of Github Actions, any thoughts?

**Comment by cnlohr** [comment]
I am not attached to Travis either way. Doesn't Github Actions cost money?

**Comment by dreua** [comment]
> I am not attached to Travis either way. Doesn't Github Actions cost money?

I don't think so, at least it has a reasonable amount of free runtime units for open source projects. We are using it to build and test PDF Arranger including Code coverage through Codecov on each PR and that works great for free.

**Comment by dreua** [comment]
Update: Github actions are free in public repositories, see https://github.com/pricing

**Comment by cnlohr** [comment]
If you would like to take ownership over this and proceed, I am def down for that.  It would be pretty awesome.  You really should get in our Discord. There's a lot of experience surrounding it.  Including @AEFeinstein .

**Comment by AEFeinstein** [comment]
I haven't worked with this project, but I have used circleci for a few others. Is that why I'm getting pinged?

**Comment by cnlohr** [comment]
yeah, just to chime in if you have any pointers.

**Comment by AEFeinstein** [comment]
Hm, nothing off the top of my head. I haven't had projects that require signing, and I'm not sure how signing key management works with circleci. That seems like the tricky part. The rest is writing a script to spin up the build environment and do the build. Easy stuff.
